utils/web3_utils.py

```diff
@@ -69,6 +69,9 @@
     Chain.SOLANA: {
         "w3": w3,
     },
+    Chain.TON: {
```

Comment:
hack: similar to the SOLANA chain, get a timestamp based on the Ethereum chain. see the `get_block_date` method below